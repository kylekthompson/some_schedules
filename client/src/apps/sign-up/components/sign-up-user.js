import PropTypes from 'prop-types';
import React from 'react';
import SignUpUserForm from 'apps/sign-up/components/sign-up-user-form';
import styled from 'styled-components';
import { Card, Container, Text } from 'components/style';
import { Separator } from 'components/form';
import { authenticated } from 'components/authentication';

const StyledContainer = styled(Container)`
  align-items: center;
  padding-top: 100px;
`;

const StyledCard = styled(Card)`
  align-items: center;
`;

export function SignUpUser({ requestSignIn }) {
  return (
    <StyledContainer>
      <StyledCard>
        <Text size={20} weight="bold">Sign Up - User</Text>
        <Separator />
        <SignUpUserForm onSuccess={requestSignIn} />
      </StyledCard>
    </StyledContainer>
  );
}

SignUpUser.propTypes = {
  requestSignIn: PropTypes.func.isRequired,
  user: PropTypes.object,
};

export default authenticated(SignUpUser);
