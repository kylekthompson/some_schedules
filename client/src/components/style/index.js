export { default as Box } from 'src/components/style/box';
export { default as Link } from 'src/components/style/link';
export { default as Text } from 'src/components/style/text';
