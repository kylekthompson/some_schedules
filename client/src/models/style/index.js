export { default as colors } from 'src/models/style/colors';
export { default as fonts } from 'src/models/style/fonts';
export { default as shadows } from 'src/models/style/shadows';
