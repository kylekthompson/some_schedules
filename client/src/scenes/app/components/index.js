export { default as Container } from 'scenes/App/components/Container';
export { default as HeaderContainer } from 'scenes/App/components/HeaderContainer';
export { default as HeaderLinks } from 'scenes/App/components/HeaderLinks';
