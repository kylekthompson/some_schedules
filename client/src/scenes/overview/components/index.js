export { default as Container } from 'scenes/Overview/components/Container';
export { default as HeaderLinks } from 'scenes/Overview/components/HeaderLinks';
export { default as Routes } from 'scenes/Overview/components/Routes';
