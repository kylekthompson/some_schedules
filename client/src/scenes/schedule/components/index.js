export { default as Container } from 'scenes/Schedule/components/Container';
export { default as ContentContainer } from 'scenes/Schedule/components/ContentContainer';
export { default as SidebarContainer } from 'scenes/Schedule/components/SidebarContainer';
