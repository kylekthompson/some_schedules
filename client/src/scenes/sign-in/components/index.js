export { default as Container } from 'scenes/sign-in/components/container';
