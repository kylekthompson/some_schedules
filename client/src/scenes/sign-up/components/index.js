export { default as Container } from 'scenes/sign-up/components/container';
