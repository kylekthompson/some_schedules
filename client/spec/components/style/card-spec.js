import React from 'react';
import { Card } from 'components/style';
import { mount } from 'spec/support/mount';

describe('<Card />', () => {
  it('is styled properly', () => {
    const { container } = mount(<Card />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
