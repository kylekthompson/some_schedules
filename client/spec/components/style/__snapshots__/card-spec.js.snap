// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> is styled properly 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 {
  box-shadow: 0 4px 6px rgba(0,61,93,0.11), 0 1px 3px rgba(12,27,42,0.08);
  background-color: rgba(255,255,255,1);
  border-radius: 4px;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  padding: 10px;
}

<div
  class="c0 c1"
/>
`;
